src/functions/binary/apt_satisfy.pl

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env perl
+my @prev_installed_packages = split('\n', `dpkg-query -Wf '\${Package}\\n'`);
+
+# Install the missing deps.
+
+if (system("apt-get", satisfy, @ARGV) != 0) {
```

Comment:
```suggestion
if (system("apt-get", "satisfy", @ARGV) != 0) {
```
Should "satisfy" be in quotes? I don't know Perl well enough to know if this is an actual typo or not

Reply:
If it's anything like Bash, the quotes don't matter. If it was a variable, it would be `$satisfy` I think. That's why it's also `@ARGV`, for an array.